app/src/modules/preferences/index.ts

```diff
@@ -0,0 +1,41 @@
+import { createSlice } from '@reduxjs/toolkit';
+import useSelector from 'hooks/useSelector';
+import { getTheme } from 'modules/selectors';
+
+export const themes = ['light' , 'dark'] as const
+export type Theme = typeof themes[number];
+
+interface IPreferences {
+  theme: Theme
+};
+
+const initialState: IPreferences = { theme: initialLoadTheme() };
```

Comment:
👍 Great idea to have preferences. This is something, that can be used in the future.